e2-studio-star-rx72n-firmware/libs/rx_crc/src/rx_dmaca.c

```
@@ -0,0 +1,1272 @@
+            break;
+        case k_dmaca_mode_block:
+            dmtmd |= (DMTMD_MD_BLOCK << DMTMD_MD_SHIFT);
+            break;
+        case k_dmaca_mode_repeat:
+            dmtmd |= (DMTMD_MD_REPEAT << DMTMD_MD_SHIFT);
+            break;
+        default:
+            return k_rx_err_nack;
+    }
+
+    if (p_cfg->data_size == k_dmaca_size_longword) {
```

Clang (14.0.6)

[warning] if with identical then and else branches

(bugprone-branch-clone)